plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -249,10 +253,6 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []v1.Volume, namespace s
 				return nil
```

Comment:
why it returns immediately whenever there is error instead of continue to check the following volumes

Reply:
That's actually a good question. `return` was there even before this PR and it looks wrong. I'll rework the PR to make it robust when a pod uses two deleted PVCs (counted as two) and two pods using the same deleted PVC (counted as one).